scalikejdbc-core/src/main/scala/scalikejdbc/ConnectionPoolSettings.scala

```diff
@@ -10,5 +10,6 @@ case class ConnectionPoolSettings(
   validationQuery: String = null,
   connectionPoolFactoryName: String = null,
   driverName: String = null,
-  warmUpTime: Long = 100L)
+  warmUpTime: Long = 100L,
+  timeZone: String = null)
```

Comment:
@tkawachi In my opinion, using `Option` here doesn't have any meaning in pactice. 
(e.g.) `ConnectionPoolSettings(driverName = Some("com.mysql.jdbc.Driver"))` instead of `ConnectionPoolSettings(driverName = "com.mysql.jdbc.Driver")` is really better?